dts/bindings/arm/nordic,nrf21540_gpio.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+
+description: >
+    This is a representation of the nRF21540 Radio Front-End module
+compatible: "nordic,nrf21540_gpio"
```

Comment:
`"nordic,nrf21540-gpio"` would probably look more in line with other `compatible` properties.